protected/humhub/modules/content/components/ContentContainerController.php

```diff
@@ -144,6 +147,30 @@ protected function checkModuleIsEnabled()
         }
     }
 
+    /**
+     * Check if the requested content record is readable for the current or given User
+     *
+     * @param ContentActiveRecord|null $record
+     * @param User|integer|null $user
+     * @return bool
+     * @throws HttpException
+     * @throws \Throwable
+     * @throws \yii\base\Exception
+     */
+    protected function checkContentIsReadable(?ContentActiveRecord $record, $user = null): bool
+    {
+        if (!($record instanceof ContentActiveRecord)) {
+            throw new NotFoundHttpException(Yii::t('base', 'The requested content cannot be displayed. Either it was deleted, you mistyped it or it is not available for you.'));
```

Comment:
@luke- instead of `not available` maybe change to `currently not available`,  this way it lets the user know that it's not available at this time but once things are fixed it may be available at a later date.

Reply:
@Semir1212 What do you think?

Reply:
Looks good. @luke-

Reply:
@yurabakhtin Can you please adapt?

Reply:
@luke- Done in commit https://github.com/humhub/humhub/pull/6005/commits/3f8754be630dfe5c9e72ed8f52586ea45b085ca4.